cpp/include/cudf/detail/utilities/device_operators.cuh

```diff
@@ -97,7 +101,7 @@ struct DeviceMin {
             typename std::enable_if_t<!std::is_same<T, cudf::string_view>::value>* = nullptr>
   static constexpr T identity()
   {
-    return std::numeric_limits<T>::max();
```

Comment:
I noticed that the duration types overload `numeric_limits` [here](https://github.com/rapidsai/cudf/blob/ce826c57c8cafb7d78f33873ca8c0c030fa42087/cpp/include/cudf/wrappers/durations.hpp#L72-L87)... but as we had discussed, that won't work for fixed point since the identities may depend on the scale factor, which would need to be passed as an additional parameter.

Reply:
Have disabled for now. Will enable once with have `scale_type` in `data_type`